### PR TITLE
make lttng_version configurable

### PR DESCRIPTION
**Pull request by razr**

### CODE REVIEW

**Review by LanderU — APPROVED** [review]
LGTM!

File: .github/workflows/build-stable.yaml [hunk 1]
```diff
@@ -8,8 +8,13 @@ on:
         required: true
         default: '5.4.0-1058-raspi'
         type: string
+      lttng_version:
+        description: 'LTTNG version e.g. 2.12'
```
Comment (LanderU) [inline .github/workflows/build-stable.yaml:12]: @razr, do we need to add here a default version as well?

Comment (razr) [reply .github/workflows/build-stable.yaml:12]: it is already added here: https://github.com/ros-realtime/linux-real-time-kernel-builder/blob/master/.github/workflows/rpi4-kernel-build.yml#L37

Comment (LanderU) [reply .github/workflows/build-stable.yaml:12]: Thank you!

**Review by carlossvg — APPROVED** [review]

**Pull request merged**